crypto/asn1/a_sign.c

```diff
@@ -131,12 +131,11 @@ int ASN1_item_sign_ex(const ASN1_ITEM *it, X509_ALGOR *algor1,
 {
     int rv = 0;
     EVP_MD_CTX *ctx = evp_md_ctx_new_ex(pkey, id, libctx, propq);
-
```

Comment:
Please just drop the changes in this file. It is unrelated now.

Reply:
Done.